lib/classes/Variables.js

```diff
@@ -822,6 +827,22 @@ class Variables {
       });
   }
 
+  getValueStrToBool(variableString) {
+    return BbPromise.try(() => {
+      const groups = variableString.match(this.strToBoolRefSyntax);
+      if (groups.length < 2) {
+        throw new this.serverless.classes.Error(`${variableString} cannot have an empty string`);
+      }
+      const param = groups[1].trim();
+      if (param === 'false') {
+        return false;
+      } else if (!param || param === '' || param === 'null') {
+        throw new this.serverless.classes.Error(`${variableString} resolved to null`);
+      }
+      return true;
```

Comment:
I'd add comment, that it's only non-empty string that can end here

Reply:
Makes sense.